pom.xml

```diff
@@ -625,6 +628,24 @@
         <artifactId>guava-testlib</artifactId>
         <version>${guava.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.easymock</groupId>
+        <artifactId>easymock</artifactId>
+        <version>${easymock.version}</version>
+        <scope>test</scope>
+        <exclusions>
+          <exclusion>
+            <groupId>org.objenesis</groupId>
+            <artifactId>objenesis</artifactId>
+          </exclusion>
+        </exclusions>
+      </dependency>
+      <dependency>
+        <groupId>org.objenesis</groupId>
+        <artifactId>objenesis</artifactId>
+        <version>${objenesis.version}</version>
+        <scope>test</scope>
```

Comment:
Since the scope for objenesis is always test, I think we should centralize it in dependency management.  We could always override it if it needs to be a different scope for a particular artifact. I think as long as the declared dependency can match the one in dependency management, it will inherit the defined scope. See [here](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html) (The dependency management section)